__tests__/index.spec.js

```diff
@@ -1,7 +1,59 @@
-// import useInputDebounce from "./../src";
+import React from "react";
+import { render, fireEvent, cleanup } from "react-testing-library";
+import useInputDebounce from "./../src";
+import { debug } from "util";
```

Comment:
Would you remove this import? you are not using it anywhere.

Reply:
removed.